src/guide/6-agent-birth-death/1-agent-birth-device.rst

```diff
@@ -1,3 +1,151 @@
 Agent Birth from Agent Functions
 ================================
 
+Agent Creation Overview
+-----------------------
+The agent creation interface usable in agent functions is only able to create a single agent per existing agent per iteration. 
+Additionally, the agent type and state being created must be earlier specified as part of the model description. If you need
+to create more than one agent for each existing agent, you should use the *TODO: link to host agent creation api*
+
+**Note:** *Agents created by agent functions do not exist until the next layer.*
+
+Enabling Agent Creation
```

Comment:
If agents have multiple states, the 2nd optional arg of the required state should be passed to `setAgentOutput()`,

https://github.com/FLAMEGPU/FLAMEGPU2/blob/1b38d4e6a7f4113fe1a00b8cd16619ec30e45ab8/include/flamegpu/model/AgentFunctionDescription.h#L164

*Just noticed this when skimming the docs, and went back to the PR*